trace-sdk/src/main/java/io/bitrise/trace/network/NetworkClient.java

```diff
@@ -116,7 +117,7 @@ static synchronized NetworkCommunicator getCommunicator() {
   @NonNull
   public static Gson getGson() {
     return new GsonBuilder()
-        .setFieldNamingStrategy(new SnakeCaseStrategy())
```

Comment:
removed the SnakeCaseStrategy class and usage here in favour of the FieldNamingPolicy. LOWER_CASE_WITH_UNDERSCORES - as it means this is code we don't have to maintain (or test). 

However, we're not actually using this line as all the info that gets sent goes through the `ProtoTypeAdapter` which has it's serializationFormat set to CaseFormat.LOWER_UNDERSCORE.  

I think it's maybe useful to keep if we add more classes in the future that we send, we don't have to change the field naming policy - what do you think?

Reply:
Lets keep it.